template-procedure.md

```diff
@@ -0,0 +1,251 @@
+# Procedure template
+
+Use this template when writing procedural (how-to) topics. Procedural topics include detailed steps
+to perform a task as well as some context about the task.
+
+
+## Template
+
+The template below includes the standard sections that should appear in procedural topics, including
+a topic sentence, an overview section, and sections for each task within the procedure.
+Copy and paste the markdown from the template to use it in your topic.
+
+This topic describes...
```

Comment:
Ehhh, maybe? It would look better in preview (not something I can get working for this doc, though) but engineers will just be looking at this in their text editors so I don't know that code tags would be more help than nuisance.

Reply:
I'll add the code tag; it's likely more help than hindrance.